### DISCUSSION TITLE
Portal-App: Parametrierter Aufruf

### DISCUSSION THREAD

**Discussion by thomaskofler** [e-netze/webgis-community#47] [General]
Hallo,

ist es möglich in der Portal-App einen parametrierten Aufruf zu machen? Ich würde nämlich gerne eine Abfrage aufrufen, damit der Nutzer direkt zum richtigen Objekt kommt und dann den Punkt dort setzen kann.

LG Thomas

**Comment by gis-eni** [comment]
Hallo Thomas,

ich weiß nicht genau, was du mit **Portal-App** meinst? Den Viewer?

Wenn ja gibts hier die Anleitung für den parametrierten Aufruf:

https://docs.webgiscloud.com/cloud/webgis_apps/mapviewer/url-parameters/index.html

man kann Abfragen mit Werten übergeben und auch gleich ein bestimmtes Werkzeug öffnen, zb das Edit Werkzeug

LG Jürgen

**Reply by thomaskofler** [reply]
Hallo Jürgen,

nein nicht den Viewer, das Datenerfassungs-Tool.
![grafik](https://user-images.githubusercontent.com/119412172/225538003-05a9df95-8665-44fd-8243-8d320aa64624.png)

Da sind alle Radrouten auf einmal ersichtlich, ich würde aber gerne die URL so parametrieren, dass ich da die Radstrecke übergebe, damit gleich auf eine bestimmte Strecke gezoomt wird.

LG Thomas

**Comment by jugstalt** [comment]
Nein, die Apps können von vorherein keine Parameter aus der Url auslesen. Man könnte nur probieren das Template entsprechend anzupassen. Wenn man dann aber noch Abfragen damit triggern möchten, wird ein bisserl komplizierte...

LG Jürgen
